migrate-contexts/README.md

```diff
@@ -0,0 +1,72 @@
+# Migrating Contexts from CircleCI On-Prem to CircleCI SaaS
+
+## Extract Contexts from On-Prem
+
+### Get List of Contexts
+1. Get onto the `services` box
```

Comment:
Is `services` the official name of the server that CircleCI On-Prem is run on? I hadn't heard this term before, so this sentence was unclear to me. Something more descriptive might be better here, like "Log onto the server that hosts your CircleCI database"

Reply:
Yeah I think the `services` box is something standard in on-prem deployments. https://circleci.com/docs/2.0/overview/#services-machine